packages/http/src/validator/validators/index.ts

```diff
@@ -1,4 +1,8 @@
-export { HttpBodyValidator } from './body';
-export { HttpHeadersValidator } from './headers';
-export { HttpQueryValidator } from './query';
-export { IHttpValidator } from './types';
+import { validate as validateBody } from './body';
+import { validate as validateHeaders } from './headers';
+import { validate as validateQuery } from './query';
+import { validate as validatePath } from './path';
+
+const obj = { validateQuery, validatePath, validateHeaders, validateBody };
+
+export = obj;
```

Comment:
```suggestion
export { validate as validateBody } from './body';
export { validate as validateHeaders } from './headers';
export { validate as validateQuery } from './query';
export { validate as validatePath } from './path';
```

^ if I'm not mistaken about the syntax..

Reply:
I tried that as well but that breaks jest spy because direct exports do not have the setter defined (and that's required to spy the function)

<img width="717" alt="image" src="https://user-images.githubusercontent.com/1416224/94038039-9f1d4b00-fd8b-11ea-8e31-3e0924ad25c4.png">

I was looking for a way to remove these tests and rewrite them, but it was too much to do at this point